tests/kafkatest/services/transactional_message_copier.py

```diff
@@ -47,12 +47,13 @@ class TransactionalMessageCopier(KafkaPathResolverMixin, BackgroundThreadService
 
     def __init__(self, context, num_nodes, kafka, transactional_id, consumer_group,
                  input_topic, input_partition, output_topic, max_messages = -1,
-                 transaction_size = 1000, enable_random_aborts=True):
+                 transaction_size = 1000, transaction_timeout = None, enable_random_aborts=True, use_group_metadata=False, group_mode=False):
```

Comment:
nit: some `=` are spaced before / after and some are not.

Reply:
Nice catch!